src/libs/reportUtils.js

```diff
@@ -24,19 +25,33 @@ function isReportMessageAttachment(reportMessageText) {
 /**
  * Given a collection of reports returns the most recently accessed one
  *
- * @param {Record<String, {lastVisitedTimestamp}>|Array<{lastVisitedTimestamp}>} reports
+ * @param {Record<String, {lastVisitedTimestamp, reportID}>|Array<{lastVisitedTimestamp, reportID}>} reports
  * @returns {Object}
  */
-function getLastAccessedReport(reports) {
+function findLastAccessedReport(reports) {
     return _.chain(reports)
         .toArray()
+        .filter(report => report && report.reportID)
         .sortBy('lastVisitedTimestamp')
         .last()
         .value();
 }
 
+/**
+ * We are decorating findLastAccessedReport so that it caches the first result once the reports load.
+ * This will ensure the initialParams are only set once for the ReportScreen.
+ */
+const getInitialReportScreenParams = _.once((reports) => {
+    const last = findLastAccessedReport(reports);
+
+    // Fallback to empty if for some reason reportID cannot be derived - prevents the app from crashing
+    const reportID = lodashGet(last, 'reportID', '');
+    return {reportID};
+});
+
```

Comment:
Moved this here as it became a bit bigger and needs to be declared above the component 

Because this is now exported - It will be an issue if this is used anywhere else but inside `MainDrawerNavigator` so maybe it should be moved there after all...